lib/utils.h

```diff
@@ -78,4 +78,7 @@ inline static const char *getOrdinalText(int i)
     return "th";
 }
 
+template<typename T>
+inline void UNUSED(T&&) {}
```

Comment:
(void)param is a common fix. However UNUSED(param) is nicer. I also personally like to comment out the argument name.

    void foo(int /* param */)